### PR TITLE
Adds header text

### PR DESCRIPTION
**Pull request by kierangillen**
- Styles city show list views and adds header text

Screen Shot 2019-03-12 at 2 55 51 PM

#trivial

### CODE REVIEW

File: src/lib/Scenes/City/CitySavedList.tsx [hunk 1]
```diff
@@ -47,18 +47,16 @@ class CitySavedList extends React.Component<Props, State> {
...
+          header="Saved shows"
+          key={name + "save"}
+          cityName={name}
+          bucket={shows.edges.map(e => e.node) as any}
+          type={"saved"}
+          relay={relay}
+          onScroll={isCloseToBottom(this.fetchData)}
+          fetchingNextPage={fetchingNextPage}
+        />
       </Theme>
```
Comment (orta) [inline src/lib/Scenes/City/CitySavedList.tsx]: This may not need the theme?

Comment (orta) [reply src/lib/Scenes/City/CitySavedList.tsx]: Ah, the one below adds it, so it probably does

File: src/lib/Scenes/City/Components/AllEvents.tsx [hunk 2]
```diff
@@ -92,7 +92,6 @@ export class AllEvents extends React.Component<Props, State> {
     if (buckets.museums && buckets.museums.length) {
       sections.push({
         type: "bmw",
-        data: buckets.museums,
```
Comment (ashfurrow) [inline src/lib/Scenes/City/Components/AllEvents.tsx]: Ah, good catch.

**Comment by kierangillen** [comment]
Something strange is happening on this branch where the components are not rendering, creating a new PR

**Pull request closed without merging**